README.md

```diff
@@ -24,8 +24,10 @@ or
 #### iOS
 
 ### Pods
-Add the line below to your podfile then run `pod install`
+1. Add the line below to your podfile then run `pod install`
 `pod 'react-native-fitness', :path => '../node_modules/@ovalmoney/react-native-fitness'`
+2. In XCode, in the project navigator, select your project. Add `libreact-native-fitness.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
+3. Add `NSHealthShareUsageDescription` to your `Info.plist` in order to ask permissions.
```

Comment:
you forgot to mention that health kit is needed is order to use it.

https://developer.apple.com/documentation/healthkit/setting_up_healthkit